### PR TITLE
fix(text): update content and grammar

### PR DESCRIPTION
**Pull request by mindreeper2420**
update the content and grammer of various paragraphs

### CODE REVIEW

File: src/app/index.html [hunk 1]
```diff
@@ -270,10 +270,10 @@ <h3 class="bullet-header">
...
+                  Allow development teams to be more confident in the technology decisions they make.
                 </h3>
                 <p>
-                  OpenShift.io helps developers make better decisions. It provides actionable recommendations and insights around the software components developers use. OpenShift.io Analytics applies machine learning algorithms based on their usage pattern of components. The data is gathered from various public data sources such as Github, Maven and NPM along with our own internal OpenShift data. Through this comprehensive dataset, developers get tangible information for credible
+                  OpenShift.io helps developers make better decisions. It provides actionable recommendations and insights around the software components developers use. OpenShift.io Analytics applies machine learning algorithms based on the usage pattern of components. The data is gathered from various public data sources such as Github, Maven and NPM along with our own internal OpenShift data. Through this comprehensive data set, developers get tangible information for credible
                 </p>
```
Comment (Preeticp) [inline src/app/index.html]: Thanks for taking care of all the points @mindreeper2420 
However, we still need a bit of content after credible 'Through this comprehensive data set, developers get tangible information for credible.....what? If it helps, I could get relevant information for this from Bayesian folks here.

Comment (mindreeper2420) [inline src/app/index.html]: @Preeticp the content was given to us by Todd and the Marketing Team, so not sure about changing anything (or adding to it). I'd suggested making changes or additions a separate story for consideration.

Comment (Preeticp) [inline src/app/index.html]: @mindreeper, If it is just about adding a few words to complete the existing content, a separate story might not be needed. I will try to follow up on this and get more information.

**Comment by Preeticp** [comment]
Just for the record, a piece of the content is still missing and ideally we should try to get it fixed.

**Comment by Preeticp** [comment]
Folks just came to know the above issue has been fixed by Adi and Adam here: https://github.com/openshiftio/openshift.io/issues/162
Thanks @mindreeper2420

**Pull request merged**